azure-communication-ui/azure-communication-ui/src/test/java/com/azure/android/communication/ui/redux/reducer/LocalParticipantReduxStateReducerUnitTest.kt

```diff
@@ -678,7 +678,7 @@ internal class LocalParticipantReduxStateReducerUnitTest {
             displayName = null
         )
         val error =
-            CallCompositeError(CommunicationUIErrorEvent.TURN_MIC_OFF, Throwable("Mic OFF failed"))
+            CallCompositeError(CommunicationUIErrorCode.TURN_MIC_OFF, Throwable("Mic OFF failed"))
```

Comment:
We use "MicOn" earlier but here I think it should be "MicOff" instead of "Mic OFF"